runtime/interpreter/src/interpreter.cpp

```diff
@@ -389,7 +389,8 @@ int handleRet(ShrimpVM *vm)
 
     std::cout << "LOG: " << instr.toString() << std::endl;
 
-    return 0;
```

Comment:
Aren't handler returns were intendent for status reports?
If so, this change looks weird

Reply:
Maybe report return value separately in case of successful interpretation?

Reply:
Changed. Now, if interpreter has ended it's work correctly, then main cycle of vm will return accumulator value